pkg/pmem-grpc/grpc.go

```diff
@@ -85,26 +85,48 @@ func LoadServerTLS(caFile, certFile, keyFile, peerName string) (*tls.Config, err
 	}
 
 	return &tls.Config{
-		MinVersion:    tls.VersionTLS12,
-		Renegotiation: tls.RenegotiateNever,
-		Certificates:  []tls.Certificate{*peerCert},
-		ClientCAs:     certPool,
-		ClientAuth:    tls.RequireAndVerifyClientCert,
-		VerifyPeerCertificate: func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
-			// Common name check when accepting a connection from a client.
-			if peerName == "" {
-				// All names allowed.
-				return nil
+		GetConfigForClient: func(info *tls.ClientHelloInfo) (*tls.Config, error) {
+			if info == nil {
+				return nil, fmt.Errorf("nil client info passed")
```

Comment:
Nit: this isn't a format string, better use `errors.New`.

Reply:
Fixed.